.travis.yml

```diff
@@ -15,4 +15,8 @@ before_script:
 script:
 - export NODE_PATH=$TRAVIS_BUILD_DIR'/privly.safariextension/privly-applications/test/node_modules'
 - cd test
-- ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js
+- "if [ ${TRAVIS_PULL_REQUEST} = 'false' ]; then ./run_each.sh ../privly.safariextension/privly-applications/test/karma.conf-ci.js; fi" #Run the unit tests
```

Comment:
No, only the extension tests are run by using the run_each.sh script in https://github.com/privly/privly-safari/blob/master/test/run_each.sh and the karma.conf-ci.js is run from privly-applications.